massa-execution-worker/src/active_history.rs

```diff
@@ -259,4 +260,19 @@ impl ActiveHistory {
             }
         }
     }
+
+    /// Get the execution statuses of operations
+    ///
+    /// # Return Value
+    ///
+    /// A hashmap with
+    /// * the operation id as the key
+    /// * and a bool as the value: true: execution succeeded, false: execution failed
+    pub fn get_op_exec_status(&self) -> HashMap<OperationId, bool> {
```

Comment:
This should use `HistorySearchResult` like the other read requests of the active history !

note: that's a huge array. 5000 tx/slot, 100 active slots = half a million elements

Reply:
#3704

Reply:
thx